src/module.js

```diff
@@ -957,6 +970,25 @@ export default class DailyIframe extends EventEmitter {
       this.validateProperties(properties);
       this.properties = { ...this.properties, ...properties };
     }
+
+    // only update if showLocalVideo/showParticipantsBar are being explicitly set
+    if (properties.showLocalVideo !== undefined) {
+      if (this._callObjectMode) {
+        console.error('showLocalVideo is not available in callObject mode');
```

Comment:
do you also want to add this check to the constructor, since these properties can be set there as well?

Reply:
or, even better, maybe you can just add this check once to the properties' validation methods in `FRAME_PROPS` and have it validated everywhere